src/diffusers/pipelines/blip_diffusion/conversion_scripts.py

```diff
@@ -0,0 +1,173 @@
+from diffusers import (
+    AutoencoderKL,
+    DDPMScheduler,
+    UNet2DConditionModel,
+)
+from transformers import CLIPTokenizer
+from src.diffusers.pipelines.blip_diffusion.modeling_blip2 import Blip2VisionModel, Blip2QFormerModel
+from src.diffusers.pipelines.blip_diffusion.modeling_ctx_clip import CtxCLIPTextModel
+from transformers.models.blip_2.configuration_blip_2 import Blip2Config, Blip2Config, Blip2VisionConfig, Blip2QFormerConfig
+from src.diffusers.pipelines import BlipDiffusionPipeline
+from clip_vit_sample import create_clip_vit_L
+from LAVIS.lavis.models import load_model_and_preprocess
```

Comment:
Hmm when we move this script to `scripts` I think we could add example usage in the comments and make it clear that `LAVIS` needs to be installed for running the script.